functions/go/apply-setters/README.md

```diff
@@ -7,9 +7,9 @@
 Apply setter values on resource fields. Setters serve as parameters for template-free
```

Comment:
This is another example of short description that needs to be clarified. It's awkward and tautological. Try avoid any use of "setter" in this sentence.

Reply:
Updated.